### PR TITLE
Added Forensic Code Analyzer and Obligation Fulfillment Mgmt to Component Model

### PR DESCRIPTION
**Pull request by misappi**
PNG not yet generated. Relationships beten Obligation Fulfillment Mgmt and Code Checker, FOSS Compliance Artefact Generator no (yet) added

### CODE REVIEW

File: Tooling-Landscape/Unanimous-Understanding/OSS-Tooling-Landscape-Glossary.md [hunk 1]
```diff
@@ -3,73 +3,102 @@
...
+
 A list of all FOSS (and commercial?) licenses in use within an organization with all necessary metadata like implied obligations. It may use a Public License Master Database and a License Obligation Database for basic information enriched by the company for internal use.
 Example: [Eclipse SW360](https://projects.eclipse.org/proposals/sw360)
 
-### Component Relationship Repository
-Software products and most software components are built from other software components. The *Component Relationship Repository* contains this relationship information, aka. *bill of materials* to enable the management of dependencies over the life cycle of the software.
-Example: [Eclipse SW360](https://projects.eclipse.org/proposals/sw360)
+### Obligation Fulfillment Management
+
+A process and/or tool set that supports the fulfillment of license obligations. This comprises company-extzernal obligations that are imposed by law as well as company-internal obligations originating in company-specific policies or processes. Typically, it utilizes Compliance Checkers and FOSS Compliance Artefact Generators.
```
Comment (blaumeiser-at-bosch) [inline Tooling-Landscape/Unanimous-Understanding/OSS-Tooling-Landscape-Glossary.md]: typo extzernal

File: Tooling-Landscape/Unanimous-Understanding/OSS_Tooling_Landscape_UMLComp.plantuml [hunk 2]
```diff
@@ -59,6 +59,13 @@ node "Company Internal" <<Company Network>> {
 
     component [Security Vulnerability Assessment] as securityassessment <<Internal Service>>
 
+    component [Forensic Code Analyzer] as forensiccodeanalyzer <<Internal Service>>
```
Comment (blaumeiser-at-bosch) [inline Tooling-Landscape/Unanimous-Understanding/OSS_Tooling_Landscape_UMLComp.plantuml]: Two comments:
1. I prefer to have the relationships after the component description like the hidden links below. Perhaps you create a section before the hidden stuff for the visual relations.
2. There are two additional use relations
`componentanalysis ..> licensescanner : use`
`componentanalysis ..> forensiccodeanalyzer : use`

File: Tooling-Landscape/Unanimous-Understanding/OSS_Tooling_Landscape_UMLComp.plantuml [hunk 3]
```diff
@@ -59,6 +59,13 @@ node "Company Internal" <<Company Network>> {
...
+
+    component [Obligation Fulfillment Management] as obligationfulfillmentmgmt <<Internal Service>>
+    
+    obligationfulfillmentmgmt ..> forensiccodeanalyzer : use
```
Comment (blaumeiser-at-bosch) [inline Tooling-Landscape/Unanimous-Understanding/OSS_Tooling_Landscape_UMLComp.plantuml]: This one is wrong. This should be:
`obligationfulfillmentmgmt ..> fossartefactgenerator : use`

**Comment by misappi** [comment]
I created a new PNG via the Plantuml online tool :-)

**Pull request merged**